pkg/cloud/services/compute/dependent_resources.go

```diff
@@ -25,7 +25,7 @@ import (
 func AdoptDependentMachineResources(scope *scope.WithLogger, baseName string, referencedResources *infrav1.ReferencedMachineResources, dependentResources *infrav1.DependentMachineResources) error {
 	networkingService, err := networking.NewService(scope)
 	if err != nil {
-		return false, err
```

Comment:
Seems like this change should be in a different commit? It's syntax error otherwise.